includes/Embed_Block.php

```diff
@@ -111,6 +108,80 @@ public function init() {
 		add_filter( 'wp_kses_allowed_html', [ $this, 'filter_kses_allowed_html' ], 10, 2 );
 	}
 
+	/**
+	 * Initializes assets needed for the block.
+	 *
+	 * Registers scripts and styles for the block itself used in the editor,
+	 * as well as the external assets used by the <amp-story-player> web component (non-AMP version).
+	 * Also registers an inline stylesheet used on the frontend to ensure correct
+	 * styling and responsiveness for the block, which is used for both the regular and the AMP version.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @return void
+	 */
+	private function init_assets() {
+		wp_register_script( 'standalone-amp-story-player', 'https://cdn.ampproject.org/amp-story-player-v0.js', [], 'v0', false );
+		wp_register_style( 'standalone-amp-story-player', 'https://cdn.ampproject.org/amp-story-player-v0.css', [], 'v0' );
+
+		$this->register_script( self::SCRIPT_HANDLE, [ 'standalone-amp-story-player', Tracking::SCRIPT_HANDLE ] );
+		$this->register_style( self::SCRIPT_HANDLE, [ 'standalone-amp-story-player' ] );
+
+		// Registering a style without a `src` allows us to just use the inline style below
+		// without needing an external stylesheet.
+		wp_register_style(
+			self::SCRIPT_HANDLE_FRONTEND,
+			'',
+			[],
+			WEBSTORIES_VERSION
+		);
+
+		$css = <<<CSS
```

Comment:
Why is this an inline style? Just wondering.

Reply:
See "Relevant Technical Choices" in the pull request description

Reply:
Can we add a `todo` or comment to explain this in the code?